pull_twitter.py

```diff
@@ -46,5 +65,28 @@
     # tweepy client
     client = Client(bearer_token=os.environ['TW_BEARER_TOKEN'], wait_on_rate_limit=True)
 
+    # Create output directories
```

Comment:
now that I actually play with this, I think it would be less cludgy to remove the data and metadata subfolders under the timestamped folder and just place the command.txt, config.yaml, and data csv all at the top level.

Reply:
Addressed in most recent commit

Reply:
Metadata and data subdirectories removed and contents placed at top level of timestamped output directories.